packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-in/go/integrationsteps.md

```diff
@@ -0,0 +1,106 @@
+## Integration steps
+
+### Step 1: Create the SDK client on application load
+
+When the user navigates to the home page and the application loads, create a new
+SDK Client object by calling the `NewClient` method.
```

Comment:
Based on the Go documentation (e.g. https://blog.golang.org/error-handling-and-go), the open and closed parenthesis (e.g. NewClient()) are not included when referring to Go functions. Adding @MikeMondragon-okta  to confirm. Mike when referring to the NewClient function, NewClient is used instead of NewClient(), correct?

Reply:
Thanks for checking! I am a JavaScript guy, and we always seem to put parens after function/method names, but I can accept that some language communities may not choose to do that.